src/Promitor.Tests.Unit/Validation/Scraper/Metrics/Sinks/StatsDMetricSinkValidationStepTests.cs

```diff
@@ -87,6 +88,38 @@ public void Validate_StatsDWithoutHost_Fails(string host)
             PromitorAssert.ValidationFailed(validationResult);
         }
 
+        [Theory]        
+        [InlineData(null)]
+        public void Validate_StatsDWithGenevaFormatWithoutGenevaConfiguration_Fails(GenevaConfiguration genevaConfiguration)
+        {
+            // Arrange
+            var runtimeConfiguration = CreateRuntimeConfiguration();
+            runtimeConfiguration.Value.MetricSinks.Statsd.MetricFormat = StatsdFormatterTypesEnum.Geneva;
+            runtimeConfiguration.Value.MetricSinks.Statsd.Geneva = genevaConfiguration;
```

Comment:
I think we can simplify it
```suggestion
        [Fact]
        public void Validate_StatsDWithGenevaFormatWithoutGenevaConfiguration_Fails()
        {
            // Arrange
            var runtimeConfiguration = CreateRuntimeConfiguration();
            runtimeConfiguration.Value.MetricSinks.Statsd.MetricFormat = StatsdFormatterTypesEnum.Geneva;
            runtimeConfiguration.Value.MetricSinks.Statsd.Geneva = null;
```